lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -138,16 +126,12 @@
     });
   };
 
-  // Makes a reasonable determination that
-  // the data passed to the function is formatted
-  // JSON and returns `true`, if so. False, if not
-
-  function isJSON (json, callback) {
-    if (!json) {
-      return callback(new Error('You forgot to pass JSON data as the first argument.'));
+  function maybeJson (json, callback) {
```

Comment:
This should not take `callback`. Functions that take `callback` should invoke it in every case, not only in the error case like here, it is confusing because it does not follow the node callback convention.

This function is just a sniffer so it should be tolerant and not throw exceptions at all. Just return false.